subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -31,6 +31,7 @@
 #define LOG_MODULE_NAME bt_ctlr_ull_df
 #include "common/log.h"
 
+#if IS_ENABLED(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
```

Comment:
For consistency use '#if defined' instead of '#if IS_ENABLED'
See also PR #32609

Reply:
Corrected. I'll create a separate PR to clean this in all DF related files (if IS_ENABLES is still available)